### PR TITLE
Fix #7338 - afll? should explain the header abbreviations

### LINKED ISSUE #7338: afll? should explain the header abbreviations
afll outputs the following fields:
> "address     size  nbbs edges    cc cost   min bound range max bound   calls locals args xref frame name"

but there is no explanation anywhere that I can find of what those fields are. Some are explanatory, but nbbs, cc, and cost in particular would be good to have a definition of available, and ideally typing afll? would show what all of the fields correspond to. Current behavior for 'afll?' is the same as 'afll'; it just ignores the '?'.

### PR DESCRIPTION
**Pull request by qrux0**

### CODE REVIEW

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -1386,8 +1409,14 @@ static int cmd_anal_fcn(RCore *core, const char *input) {
 			eprintf ("afil - verbose function info\n");
 			eprintf ("afi* - function, variables and arguments\n");
 			break;
-		case 'j':   // "afij"
 		case 'l':   // "afil"
+			if (input[3] == '?') {
+				help_msg_afll[1] = "afil";
+				r_core_cmd_help(core, help_msg_afll);
```
Comment (radare) [inline libr/core/cmd_anal.c]: missing space before (

File: libr/core/cmd_anal.c [hunk 2]
```diff
@@ -1386,8 +1409,14 @@ static int cmd_anal_fcn(RCore *core, const char *input) {
 			eprintf ("afil - verbose function info\n");
 			eprintf ("afi* - function, variables and arguments\n");
 			break;
-		case 'j':   // "afij"
 		case 'l':   // "afil"
+			if (input[3] == '?') {
+				help_msg_afll[1] = "afil";
```
Comment (radare) [inline libr/core/cmd_anal.c]: wat, why do you need to modify this?

Comment (radare) [inline libr/core/cmd_anal.c]: i mean. if you modify this you'll have to modify it back after using it

**Comment by qrux0** [comment]
@radare Thanks for feedback.

Missing space restored.

*afil* and *afll* commands have similar help message and  __help_msg_afll[1]__ assignment at 1414 and 1447 lines sets current cmd name.
I don't have to restore it, because I modify it every time before __r_core_cmd_help (core, help_msg)__ call.

**Pull request merged**